cookbooks/clearwater/templates/default/shared_config.erb

```diff
@@ -10,6 +10,22 @@ sas_server=<%= @node[:clearwater][:sas_server] or "0.0.0.0" %>
 enum_server=<%= @node[:clearwater][:enum_server] or "" %>
 alias_list=<%= @alias_list %>
 
+<% if not @database.nil? %>
```

Comment:
Why not just 'if @database' (or at least if @database.nil) - it's easier to read